package.json

```diff
@@ -20,9 +20,10 @@
     "lint": "eslint . && format-message lint src/**/*.js",
     "prepublish": "in-publish && npm run build || not-in-publish",
     "start": "webpack-dev-server",
-    "tap": "tap ./test/{unit,integration}/*.js",
+    "tap": "tap ./test/{unit,integration,execute}/*.js",
     "tap:unit": "tap ./test/unit/*.js",
-    "tap:integration": "tap ./test/integration/*.js",
+    "tap:execute": "tap ./test/execute/*.js",
```

Comment:
I'm not sure if execute should have its own `tap:*` command. Should execute tests be inside the integration folder somehow?